### PR TITLE
[ADD] New module *email_template_cron*

### PR DESCRIPTION
**Pull request by oihane**

### CODE REVIEW

File: email_template_cron/data/email_template_cron_data.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+            <field name="interval_number">1</field>
+            <field name="interval_type">hours</field>
+            <field name="numbercall">-1</field>
+            <field eval="False" name="doall"/>
+            <field eval="'email.template'" name="model"/>
+            <field eval="'process_automatic_tmpl_queue'" name="function"/>
+            <field eval="'()'" name="args"/>
+        </record>
+    </data>
+</openerp>
```
Comment (Daniel-CA) [inline email_template_cron/data/email_template_cron_data.xml]: Te falta return

File: email_template_cron/views/email_template_view.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+                        <field name="interval_type" />
+                        <newline/>
+                        <field name="nextcall" />
+                        <field name="cron_domain" />
+                    </group>
+                </page>
+            </field>
+        </record>
+    </data>
+</openerp>
```
Comment (Daniel-CA) [inline email_template_cron/views/email_template_view.xml]: return

**Review by Daniel-CA — APPROVED** [review]

**Comment by Daniel-CA** [comment]
:+1:

**Review by esthermm — APPROVED** [review]
:+1:

**Comment by anajuaristi** [comment]
:+1:

**Pull request merged**